app.js

```diff
@@ -1,13 +1,17 @@
 const express = require('express');
 const app = express();
+app.use(express.json())
```

Comment:
You don't need this until you are sending a body with your request, such as for PATCH or POST